src/components/toaster/toast/toast.module.scss

```diff
@@ -0,0 +1,58 @@
+@import '@styles/variables.scss';
+
+.root {
+    position: absolute;
+    top: var(--headerHeight);
+    padding: 14px;
+    width: 480px;
+    color: var(--white);
+    pointer-events: all;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
+    gap: 14px;
+}
+
+.root.inCart {
+    inset: auto 0 0 auto;
+    width: 100%;
+    max-width: var(--drawerMaxWidth);
+}
+
+@media (max-width: $desktop-width) {
+    .root {
+        top: auto;
+        bottom: 0;
+        width: 100%;
```

Comment:
So it will be wider on tablets than on desktop?

Reply:
yes. The same as in Wix Studio template